webwrapper/utils/path_builder.py

```diff
@@ -0,0 +1,19 @@
+from inspect import signature
+from inspect import Parameter
+
+
+def build_path(func, base_name: str = None, param_kinds: list = None) -> str:
+    if not callable(func):
+        raise TypeError(f'{func} is not a callable object.')
+    if base_name is None:
+        base_name = func.__name__
+    if param_kinds is None:
+        param_kinds = [Parameter.POSITIONAL_OR_KEYWORD, Parameter.KEYWORD_ONLY]
+
+    path = base_name
+    sig = signature(func)
+    for param in sig.parameters.values():
+        if (param.kind in param_kinds and
+                param.default is param.empty):
+            path = path + "/{{{0}}}".format(param.name)
```

Comment:
fastAPI support assigning type to the path params as well. for example:
path/{name:str}/{age:int}

the type of the arguments are available from `param.annotation` attribute, we want to include those in the path is the function defines them.

that will allow us to differentiate between overloaded endpoints that take different types of arguments. for example:
path/{name:str}/{age:int}
and
path/{name:str}/{age:str}

Reply:
done